test/meta-colony.js

```diff
@@ -60,23 +73,25 @@ contract("Meta Colony", accounts => {
       expect(newSkill.nParents).to.eq.BN(1);
       expect(newSkill.nChildren).to.be.zero;
 
-      // Check rootSkill.nChildren is now 1
+      // Check nChildren of the skill corresponding to the root domain in the metacolony is now 2
       const rootSkill = await colonyNetwork.getSkill(1);
-      expect(rootSkill.nChildren).to.eq.BN(1);
+      expect(rootSkill.nChildren).to.eq.BN(2);
 
       // Check rootSkill.children first element is the id of the new skill
       const rootSkillChild = await colonyNetwork.getChildSkillId(1, 0);
-      expect(rootSkillChild).to.eq.BN(4);
+      const rootSkillChild2 = await colonyNetwork.getChildSkillId(1, 1);
+      expect(rootSkillChild).to.eq.BN(2);
+      expect(rootSkillChild2).to.eq.BN(4);
     });
 
     it("should not allow a non-founder role in the metacolony to add a global skill", async () => {
-      await checkErrorRevert(metaColony.addGlobalSkill(1, { from: OTHER_ACCOUNT }), "ds-auth-unauthorized");
+      await checkErrorRevert(metaColony.addGlobalSkill({ from: OTHER_ACCOUNT }), "ds-auth-unauthorized");
     });
 
     it("should be able to add multiple child skills to the root global skill", async () => {
```

Comment:
Can we rename that test to maybe `should be able to add multiple child skills to the root domain skill`